v03_pipeline/lib/misc/io.py

```diff
@@ -108,6 +111,9 @@ def import_callset(
         mt = split_multi_hts(mt)
     if dataset_type == DatasetType.SV:
         mt = mt.annotate_rows(variant_id=mt.rsid)
+    if sample_type == SampleType.WES and filters_path:
```

Comment:
is there a reason to restrict this to exome data? If currently we only do this for WES data but not WGS, we could just only provide the `filters_path` for WES but leave open the option in the future to use this more broadly